docs/apache-airflow/faq.rst

```diff
@@ -522,3 +522,27 @@ This means ``explicit_defaults_for_timestamp`` is disabled in your mysql server
 
 #. Set ``explicit_defaults_for_timestamp = 1`` under the ``mysqld`` section in your ``my.cnf`` file.
 #. Restart the Mysql server.
+
+Does Airflow collect any telemetry data?
+----------------------------------------
+
+.. _airflow-telemetry-faq:
+
+Airflow integrates `Scarf <https://about.scarf.sh/>`__ to collect basic telemetry data during operation.
+This data assists Airflow maintainers in better understanding how Airflow is used.
+Insights gained from this telemetry are critical for prioritizing patches, minor releases, and
+security fixes. Additionally, this information supports key decisions related to the development road map.
+
+Deployments can opt-out of analytics by setting the :ref:`[telemetry_collection] enabled <config:telemetry_collection__enabled>`
+option to ``False``, or the ``SCARF_ANALYTICS=false`` environment variable.
+Individual users can easily opt-out of analytics in various ways documented in the
+`Scarf Do Not Track docs <https://docs.scarf.sh/gateway/#do-not-track>`__.
+
+The telemetry data collected is limited to the following:
+
+- Airflow version
+- Python version
+- Operating system & machine architecture
+- Executor
+- Metadata DB type & their version
```

Comment:
nit: 
```suggestion
- Metadata DB type & its version
```

Reply:
fixed in https://github.com/apache/airflow/pull/39510/commits/93c94e6b48a93787654ff4c712681ed3945060fe